src/lantmateriet/geometry.py

```diff
@@ -235,6 +235,19 @@ def __init__(self, file_path: str, detail_level: str, layer: str, use_arrow: boo
                 f"The level of detail: {detail_level} is not implemented."
             )
 
+        self._file_path = file_path
+        self._layer = layer
+        self._name = name
+        self._field = field
+
+        self.df = gpd.read_file(
```

Comment:
is it possible to use another naming than df, (same comment for class above.)

Reply:
Do you have a proposal? We have used df for dataframes in other packages. `geodata`?

Reply:
geometry?

Reply:
Perhaps the class should be named `Vector` and the field could then be `geometry`. Can we rename in another PR?

Reply:
df or data is also ok.